tasks/config.py

```diff
@@ -16,6 +16,7 @@ class Aws(BaseModel, extra=Extra.forbid):
             keyPairName: Optional[str]
             publicKeyPath: Optional[str]
             privateKeyPath: Optional[str] = None
+            privateKeyPassword: Optional[str] = None
```

Comment:
If we store the private key password in this file, we should explicitly generate a warning in `setup.py` to use a key dedicated to test accounts.